styles/helpers/_borders.scss

```diff
@@ -7,12 +7,8 @@
   );
 }
 
-.pseudo-left-border-blue-2px {
-  @include before-left-border(
-    2px,
-    solid,
-    theme('colors.blue.primary')
-  );
+.pseudo-border-blue-2px-no-border-radius-on-focus {
```

Comment:
I also need this for the revisions view. This pattern is starting to get a bit smelly.